algorithms/CPlusPlus/Maths/prime-number.cpp

```diff
@@ -0,0 +1,31 @@
+// It is a prime number program in c language.
+// it is used to check whether a given number is prime or not
+
+#include <iostream>				//header
+using namespace std;
+int main() {					//main function
+	int num, i, a = 0;			//define variable
+	cin >> num;				//taking input from user
+	
+	for(i = 2; i <= num/2; i++) {		 //for loop started
+		if (num % i == 0){		 //if condition
+		    a++;
+		}
+	}
+	if(num < 2) {
+	   a++;
+	}
+	if(a == 0){
+  	   cout << "Prime";		  // printing if the number is prime
+	}
+	else {
+	   cout << "Non-Prime";			  // printing if the number is non-prime
+	}
+	return 0;		  		  // returning from main function
+}
+//	sample input   ||	sample output
+//	      1        ||        Nom-Prime
+//	      2        ||          Prime
+//            3        ||          Prime
```

Comment:
This is odd make it proper and I told you to add the Time Complexity of the Program as well as to use [Code Formatter](https://codebeautify.org/cpp-formatter-beautifier) to give proper identation 
This is not what I want